src/options.ts

```diff
@@ -34,24 +34,13 @@ export interface BaseCapacitorOptions{
   attachThreads?: boolean;
 
   /**
-  * Enables Out of Memory Tracking for iOS and macCatalyst.
-  * See the following link for more information and possible restrictions:
-  * https://docs.sentry.io/platforms/apple/guides/ios/configuration/out-of-memory/
-  *
-  * @default true
-  * */
+   * Enables Out of Memory Tracking for iOS and macCatalyst.
+   * See the following link for more information and possible restrictions:
+   * https://docs.sentry.io/platforms/apple/guides/ios/configuration/out-of-memory/
```

Comment:
Let's point to the new docs page.

```suggestion
   * https://docs.sentry.io/platforms/apple/guides/ios/configuration/watchdog-terminations/
```